.github/workflows/pull-request.yml

```
@@ -0,0 +1,56 @@
+     - name: "Run docker build and doker push"
+       run:  echo "Tag for image ${GITHUB_REF##*/}.${RUN_NUM}"    
+     - name: Log in to Docker Hub
+       uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+       with:
+         username: ${{ secrets.DOCKER_USERNAME }}
+         password: ${{ secrets.DOCKER_PASSWORD }}
+     - name: Extract metadata for Docker
+       id: meta
+       uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+       with:
+          images: |
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 9 but found 10

(indentation)